.ci/register_package.jl

```diff
@@ -56,7 +57,9 @@ mktempdir() do download_dir
     BinaryBuilder.rebuild_jll_package(merged; download_dir=download_dir, upload_prefix=upload_prefix, verbose=verbose, lazy_artifacts=lazy_artifacts)
     
     # Upload them to GitHub releases
-    BinaryBuilder.upload_to_github_releases(repo, tag, download_dir; verbose=verbose)
+    should_upload && BinaryBuilder.upload_to_github_releases(repo, tag, download_dir; verbose=verbose)
```

Comment:
I think we need to alter `rebuild_jll_package()` here to build a JLL package that points to the BB build cache, rather than to github releases.  Otherwise, it doesn't know how to download the artifacts.

The only reason this worked in your PR so far, I believe, is because the artifacts already exist on disk because they were built there.  ;)

Reply:
Aren't we downloading the packages from the cache in the lines above? But I was thinking of pointing it directly to the cache.

Reply:
Yes, but we don't want to; that's downloading the tarballs as files, not downloading them and unpacking them as artifacts.  We need that to happen in order for JLL code to pick up the libraries.  The easiest way for us to do that is probably to just unpack the relevant tarball (just the one for the current platform; don't need to download them all) into the proper place for the artifact.  Then, because the artifact already exists locally, the JLL won't try to download it from JuliaBinaryWrappers, where it doesn't exist yet.

To unpack it, you can do something like this:

```
create_artifact() do dir
    unpack(tarball, dir)
end
```